.circleci/config.yml

```diff
@@ -1099,6 +1099,10 @@ workflows:
             - b_bytecode_osx
             - b_bytecode_ems
 
+      # Code Coverage enabled build and tests
+      - b_ubu_codecov: *workflow_trigger_on_tags
+      - t_ubu_codecov: *workflow_ubuntu2004_codecov
+
   nightly:
```

Comment:
I see that there's already a nightly run of codecov. Where does that submit its report? Maybe it should be configured to post to gitter?